fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -592,7 +593,7 @@ public void testPatchBinaryDescription() throws Exception {
         patch.addHeader("Content-Type", "application/sparql-update");
         final BasicHttpEntity e = new BasicHttpEntity();
         e.setContent(new ByteArrayInputStream(
-                ("INSERT { <" + location +
+                ("INSERT { <" + serverAddress + pid + "/x" +
```

Comment:
This integration test update would indicate that the logic:
https://github.com/fcrepo4/fcrepo4/pull/836/files#diff-d372035f89c8d0846920e20bd13e0145R591

...Should be `if (resource instanceof NonRdfSource)`